web/src/components/ShelterInformationSection.tsx

```diff
@@ -38,10 +37,10 @@ const TitleHint = styled.span`
   color: ${props => props.theme.colors.textSecondaryColor};
 `
 
-const Detail = styled.div<{ extended: boolean; to?: string }>`
+const Detail = styled.div<{ extended: boolean; elevated: boolean }>`
   padding: 5px 10px;
   display: flex;
-  ${props => (props.extended ? 'width: 100%;' : 'width: 220px;')}
+  ${props => (props.elevated ? 'cursor: pointer;' : '')}
```

Comment:
What has elevation style to do with which cursor to use? :see_no_evil:

Reply:
the stuff that is elevated is clickable and should show a cursor for this

Reply:
sorry fixed